src/components/floating/DefaultCustomContextMenu.tsx

```diff
@@ -0,0 +1,56 @@
+import Button from '../Button'
+import CustomContextMenu, { CustomContextMenuProps } from './CustomContextMenu'
+
+type DefaultContextMenuProps = {
+  menus: { text: string; icon?: React.ReactNode; onClick: () => void }[]
+  closeMenu: () => void
+}
+export type DefaultCustomContextMenuProps = Omit<
+  CustomContextMenuProps,
+  'menuPanel'
+> &
+  Omit<DefaultContextMenuProps, 'closeMenu'>
+
+export default function DefaultCustomContextMenu({
```

Comment:
For me it is strange name DefaultCustom :)

Maybe better use Common instead of default?

Reply:
Renamed in [7831237](https://github.com/dappforce/grillchat/pull/38/commits/78312379b12f24e6a2e52f60c7090f0730db5261)